docs/crocks/Pair.md

```diff
@@ -0,0 +1,85 @@
+# Pair
+
+`Pair c a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Pair` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`of`](#of), [`type`](#type) | [`ap`](#ap), [`bimap`](#bimap), [`chain`](#chain), [`concat`](#concat), [`equals`](#equals), [`fst`](#fst), [`inspect`](#inspect),  [`map`](#map), [`merge`](#merge), [`of`](#of), [`type`](#type), [`snd`](#snd), [`swap`](#swap), [`value`](#value) |
+
+## Constructor
+
+### of
+
+`Pair m => c a -> m c a`
```

Comment:
Do not need `of`, it is going away but it would have been:
`Pair m, Unit u => a -> m u a`